instance/models/openedx_instance.py

```diff
@@ -111,10 +115,7 @@ def domain(self):
 
         Returns external domain if present, otherwise falls back to internal domain.
         """
-        if self.external_lms_domain:
-            return self.external_lms_domain
-        else:
-            return self.internal_lms_domain
+        return self.get_domain('lms')
```

Comment:
@haikuginger Why?  It would just increase the level of indirection, and I can't see what we would gain.